cfme/tests/ansible/test_embedded_ansible_manual.py

```diff
@@ -974,3 +973,141 @@ def test_automate_ansible_playbook_method_type_verbosity():
         tags: ansible_embed
     """
     pass
+
+
+@pytest.mark.tier(2)
+def test_embed_tower_repo_add_remote_zone():
+    """
+    Test whether repository or credentials on New Zone.
+
+    Bugzilla:
+        1656308
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: high
```

Comment:
optional: 
fyi `high` is default caseimportance.

Reply:
Removed.